.github/workflows/check.yml

```diff
@@ -292,6 +292,8 @@ jobs:
                   arguments: ":plugin:verifyPlugin"
 
             - name: Check bundled Rust formatters
+              # FIXME: Drop this line after fixing https://github.com/intellij-rust/intellij-rust/issues/8843
+              if: matrix.os != 'windows-latest' || matrix.rust-version != 'nightly-2022-05-15'
```

Comment:
It will fail if we update nightly version. Maybe [startsWith](https://docs.github.com/en/actions/learn-github-actions/expressions#startswith) function help you here?